core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1444,7 +1180,8 @@ oppia.factory('autosaveInfoModalsService', [
             };
 
             $scope.lostChangesHtml = (
-              lostChangesService.makeHumanReadable(lostChanges).html());
+              ChangesInHumanReadableFormService.makeHumanReadable(
+                  lostChanges).html());
```

Comment:
Deindent by 2 spaces.

Reply:
@darthgera123, sorry for not making it clear before but I meant deindent line 1184 by 2 spaces and not line 1183!